src/zenml/integrations/kubeflow/orchestrators/kubeflow_orchestrator.py

```diff
@@ -77,17 +80,120 @@ class KubeflowOrchestrator(BaseOrchestrator):
 
     custom_docker_base_image_name: Optional[str] = None
     kubeflow_pipelines_ui_port: int = DEFAULT_KFP_UI_PORT
-    kubernetes_context: Optional[str] = None
+    kubernetes_context: str = ""
     synchronous = False
 
     # Class Configuration
-    FLAVOR: ClassVar[str] = "kubeflow"
+    FLAVOR: ClassVar[str] = KUBEFLOW
+
+    @staticmethod
+    def _get_k3d_cluster_name(uuid: UUID) -> str:
+        """Returns the K3D cluster name corresponding to the orchestrator
```

Comment:
```suggestion
        """Returns the k3d cluster name corresponding to the orchestrator
```

Unsure about this, but previously I've usually seen it all with lowercase in how it's referenced inside ZenML.